data/mcnp/README.md

```diff
@@ -0,0 +1,25 @@
+# MCPN codes
+
+## Binary files with scored particles
+
+Following binary files contain the list of particles scored at a plane close to the particle source. Each of the files is 2GB in size and contains roughly 10^8 particles.
```

Comment:
can you be a bit more specific? -49.9 cm upstream from isocenter?

Reply:
I can address that comment in a separate PR as it refers to the file which was not modified here.